### ISSUE TITLE
Feature request - Number field filters, allow single input field rather than range

### ISSUE THREAD

**Issue by anchovy** [craft-snippets/Craft-quick-filters#11]
We've got some custom fields that are 'numbers' - when adding a filter for these fields it renders 'from' and 'to' input fields:
![image](https://github.com/craft-snippets/Craft-quick-filters/assets/599005/d60707ca-92b5-46dd-aebc-b288c190278a)

In our use case we just want to filter on a specific number and not a range:
![image](https://github.com/craft-snippets/Craft-quick-filters/assets/599005/9cd19ca2-f69f-404a-a98c-d191e1c6aaf2)

Could the number field type filter have an option - 'Single number' or 'Range'?
![image](https://github.com/craft-snippets/Craft-quick-filters/assets/599005/cc2e079a-1892-452f-9a7e-223ba40dbe99)

**Comment by piotrpog** [comment]
Good idea. However due to large amount of work i will not be able to include it right now, probably within something like one month.

**Comment by piotrpog** [comment]
Hi @anchovy 
sorry that it took so long, but with version 3.1.0, this is now possible.
